### DISCUSSION TITLE
Password field not getting expanded in Mac and documentation is for old versions of Mac

### DISCUSSION THREAD

**Discussion by kanlanc** [espanso/espanso#1767] [General]
The Autofix in the documentation seems to be outdated and for the latest macOs, I dont see anything in automation that I can add an application.

Has anyone been able to fix this problem?

**Comment by smeech** [comment]
Discussion on [Discord](https://discord.gg/4QARseMS6k) suggests the fix probably only applies to the Catalina, Big Sur, and Monterey versions of MacOS.
Are you still having problems? Does it involve a password manager.

**Comment by federico-terzi** [comment]
@kanlanc Unfortunately, Espanso will never be able to work with password fields on macOS, that's a limitation/security protection from the Operating System itself:

<img width="910" alt="image" src="https://github.com/espanso/espanso/assets/20475953/f109cc50-ce99-4a03-9064-fb7565834a51">
